pyfar/signals/deterministic.py

```diff
@@ -255,6 +385,476 @@ def exponential_sweep_time(n_samples, frequency_range, n_fade_out=90,
     return signal
 
 
+def exponential_sweep_freq(
+        n_samples, frequency_range, start_margin, stop_margin, fade_in=None,
+        fade_out=None, butterworth_order=8, sampling_rate=44100):
+    """
+    Generate single channel sine sweep with exponentially increasing frequency.
+
+    Sweep sweep synthesis according to [#]_.
```

Comment:
```suggestion
    Sine sweep synthesis according to [#]_.
```

Reply:
Stupid me - it should really be fixed now!